### PR TITLE
fix: sync person documents timestamps with person's

### PR DESCRIPTION
**Pull request by progsmile**
edenlabllc/ehealth.api#3978

### CODE REVIEW

File: apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  alias Ecto.Adapters.SQL
+  alias Ecto.UUID
+
+  import Ecto.Query
+
+  @disable_ddl_transaction true
+  @batch_size 2000
+
+  def change do
+    SQL.query(
```
Comment (AlexKovalevych) [inline apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs]: why not just `execute`?

File: apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  use Ecto.Migration
+
+  alias Core.Repo
+  alias Ecto.Adapters.SQL
+  alias Ecto.UUID
+
+  import Ecto.Query
+
+  @disable_ddl_transaction true
+  @batch_size 2000
```
Comment (AlexKovalevych) [inline apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs]: may be too many, i would start with 100

Comment (progsmile) [reply apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs]: ok

File: apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs (hunk 1 above)
Comment (progsmile) [reply apps/core/priv/repo/migrations/20190416134539_update_person_documents_timestamps.exs]: When use `execute`, `person_documents_update_temp` is not ready for quering at this point
https://github.com/edenlabllc/mpi.api/pull/300/files/76f1aa39ae09611cbcefcf1d1901f35e22d6fc89#diff-ea2595063ef977c0584353c2bb98c11dR25

**Pull request merged**